resources/x509_cert_gen/openssl-san.cnf

```diff
@@ -10,4 +10,4 @@ keyUsage = nonRepudiation, digitalSignature, keyEncipherment
 subjectAltName = @alt_names
 
 [ alt_names ]
-DNS.1 = *.couchbase.com
+DNS.1 = s10501-ip6.qe.couchbase.com
```

Comment:
this file is not supposed to checkin. This is dynamically added. Can you delete this commit?